src/encoded/static/components/view_controls.js

```diff
@@ -309,27 +315,41 @@ const BATCH_DOWNLOAD_PROHIBITED_PATHS = [
  * Displays the modal to initiate downloading files. This modal gets displayed unconditionally in
  * this component, so parent components must keep state on whether this modal is visible or not.
  */
-export const BatchDownloadModal = ({ additionalContent, disabled, downloadClickHandler, closeModalHandler }) => (
+export const BatchDownloadModal = ({ additionalContent, disabled, downloadClickHandler, closeModalHandler, resultTotal }) => (
     <Modal focusId="batch-download-submit" closeModal={closeModalHandler} >
         <ModalHeader title="Using batch download" closeModal={closeModalHandler} />
         <ModalBody>
-            <p>
-                Click the &ldquo;Download&rdquo; button below to download a &ldquo;files.txt&rdquo; file that contains a list of URLs to a file containing all the experimental metadata and links to download the file.
-                The first line of the file has the URL or command line to download the metadata file.
-            </p>
-            <p>
-                Further description of the contents of the metadata file are described in the <a href="/help/batch-download/">Batch Download help doc</a>.
-            </p>
-            <p>
-                The &ldquo;files.txt&rdquo; file can be copied to any server.<br />
-                The following command using cURL can be used to download all the files in the list:
-            </p>
-            <code>xargs -L 1 curl -O -J -L &lt; files.txt</code><br />
+            {resultTotal < MAX_DOWNLOADABLE_RESULT ?
+                <>
+                    <p>
+                        Click the &ldquo;Download&rdquo; button below to download a &ldquo;files.txt&rdquo; file that contains a list of URLs to a file containing all the experimental metadata and links to download the file.
+                        The first line of the file has the URL or command line to download the metadata file.
+                    </p>
+                    <p>
+                        Further description of the contents of the metadata file are described in the <a href="/help/batch-download/">Batch Download help doc</a>.
+                    </p>
+                    <p>
+                        The &ldquo;files.txt&rdquo; file can be copied to any server.<br />
+                        The following command using cURL can be used to download all the files in the list:
+                    </p>
+                    <code>xargs -L 1 curl -O -J -L &lt; files.txt</code><br />
+                </>
+            :
+                <>
+                    <p>
+                        This search is too large (&gt;500 datasets) to automatically generate a manifest or metadata file.  We are currently working on methods to download from large searches.
```

Comment:
It’d be nice to insert the MAX_DOWNLOADABLE_RESULT constant so that if you change that, this text automatically changes correctly.